libcudacxx/include/cuda/std/detail/libcxx/include/__config

```diff
@@ -865,21 +865,22 @@ typedef unsigned int char32_t;
 
 // NVRTC has a bug that prevented the use of delegated constructors, as it did not accept execution space annotations.
 // This creates a whole lot of boilerplate that we can avoid through a macro (see nvbug3961621)
-#  if _CCCL_COMPILER(NVRTC) || (_CCCL_CUDACC_BELOW(11, 3) && _CCCL_COMPILER(CLANG))
-#    define _LIBCUDACXX_DELEGATE_CONSTRUCTORS(__class, __baseclass, ...)                               \
-      using __base = __baseclass<__VA_ARGS__>;                                                         \
-      template <class... _Args, enable_if_t<_CCCL_TRAIT(is_constructible, __base, _Args...), int> = 0> \
-      _LIBCUDACXX_HIDE_FROM_ABI constexpr __class(_Args&&... __args) noexcept(                         \
-        _CCCL_TRAIT(is_nothrow_constructible, __base, _Args...))                                       \
-          : __base(_CUDA_VSTD::forward<_Args>(__args)...)                                              \
-      {}                                                                                               \
+#  if (_CCCL_CUDACC_BELOW(12, 6) && _CCCL_COMPILER(NVRTC)) || (_CCCL_CUDACC_BELOW(11, 3) && _CCCL_COMPILER(CLANG))
```

Comment:
@leofang Here you can see that the workaround is in place for NVRTC and CTK < 12.6 as well as clang and CTK below 11.3

Reply:
Great, thanks for the pointer Michael!